### PR TITLE
Added Time and Space Complexity to Kadane's Algo

### PR DESCRIPTION
**Pull request by LouiseBC**

### CODE REVIEW

File: MaximumSubarray/Kadane.cpp [hunk 1]
```diff
@@ -1,7 +1,16 @@
 /*
     Kadane's algorithm:
+    ---------------
```
Comment (faheel) [inline MaximumSubarray/Kadane.cpp]: Almost there! 😄

File: MaximumSubarray/Kadane.cpp [hunk 2]
```diff
@@ -1,7 +1,16 @@
...
     Used for finding the contiguous subarray within a one-dimensional array
     of integers which has the largest sum.
+    
+    Time complexity
+    ---------------
+    O(N), where N is the number of elements in the original array.
+    
+    Space Complexity
```
Comment (faheel) [inline MaximumSubarray/Kadane.cpp]: `C` in `Complexity` should be lower-case.

**Review by faheel — COMMENTED** [review]
Just a few minor changes required before this can be merged

**Comment by LouiseBC** [comment]
Ok, I've changed it :)

**Comment by faheel** [comment]
@LouiseBC Thanks for your help!

**Pull request merged**